src/bot/global-shortcut.ts

```diff
@@ -51,6 +51,42 @@ app.shortcut("poll/create-poll-form", async ({ shortcut, ack }) => {
                         initial_value: "Option 1\nOption 2",
                     },
                 },
+                {
+                    block_id: 'deadlineDate',
+                    type: "section",
+                    text: {
+                        type: "mrkdwn",
+                        text: "Pick a date for the poll deadline."
+                    },
+                    accessory: {
+                        type: "datepicker",
+                        initial_date: "2021-12-17",
```

Comment:
What about having 1 day later from now always as the `initial_date` and the same behaviour for the `initial_time`?
Also, I wonder if it complaint with different timezones.